src/Propel/Generator/Reverse/MysqlSchemaParser.php

```diff
@@ -283,6 +296,74 @@ public function getColumnFromRow($row, Table $table)
         return $column;
     }
 
+    /**
+     * Sets table and column descriptions according to source.
+     *
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return \Propel\Generator\Model\Table
+     */
+    protected function addDescriptions(Table $table): Table
+    {
+        $tableDescription = $this->loadTableDescription($table);
+        if ($tableDescription) {
+            $table->setDescription($tableDescription);
+        }
+
+        foreach ($table->getColumns() as $column) {
```

Comment:
I would move this to the separate method

Reply:
Why though? Doesn't seem to make a big difference.
Changed it anyway. Happiness achieved, I hope.

Reply:
Yep, thank you!